core/src/main/java/org/frankframework/util/XmlBuilder.java

```diff
@@ -122,47 +122,43 @@ public void setValue(String value, boolean encode) {
 		parseText = !encode && XmlUtils.isWellFormed(value);
 	}
 
-	public String toXML() {
-		return toXML(false);
+	public Message asMessage() {
+		return new Message(asXmlString());
```

Comment:
Van mij zou je zelfs contentype text/xml of applicaiton/xml mogen zetten, en de deprecated tag kan hier misschien beter vanaf.. er is geen alternatief, we schieten er niets mee op.